CourseDevelopment/Week1/CODE_OF_CONDUCT_template.md

```diff
@@ -0,0 +1,60 @@
+# CODE OF CONDUCT 2022
+
+This code of conduct will been decided by cohort participants and is enacted from DDMMYYYY.
+The procedure is based on [this exercise by Aurelia Moser](http://aureliamoser.com/aaas-guides/conduct/index.html)  
+
+Everyone taking part in the course and discussions (mentors, helpers, coordinators, and learners) is required to conform to the following Code of Conduct. Coordinators will oversee adherence to this code throughout the course.
+ 
+#### Characteristics we value:  
+
+
+#### Behaviors we encourage
+
+
+#### Behaviors we discourage? 
+
+	
+#### We make each others feel safe and supported by
+
+    
+#### How to report an issue, should someone violate the code? 
+
+1. Contact Sanli or Erik by private message, or in person. All communication will be treated as confidential. 
```

Comment:
```suggestion
1. Contact Sanli Faez by private message, or in person. All communication will be treated as confidential. 
```

I suggest that you as coordinator are first point of contact (as also described at the bottom of this document)